### ISSUE TITLE
Failure to open Windows Event Log results in Fluentd crash

### ISSUE THREAD

**Issue by qingling128** [fluent/fluent-plugin-windows-eventlog#33]
[Version]
fluent-plugin-windows-eventlog-0.2.1

[Related Code]
When we failed to open Windows Event log file for some reason, the code still tried to close the `nil` object. Then Fluentd crashed.
https://github.com/fluent/fluent-plugin-windows-eventlog/blob/02ff79f307afa0546363eb4b41dd7c58117cbd1a/lib/fluent/plugin/in_windows_eventlog.rb#L184

[Error]
```
2019-08-18 07:13:24 -0500 [error]: #0 Unexpected error raised. Stopping the timer. title=:in_windows_eventlog_security error_class=NoMethodError error="undefined method `close' for nil:NilClass\nDid you mean?  clone"
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluent-plugin-windows-eventlog-0.2.1/lib/fluent/plugin/in_windows_eventlog.rb:184:in `ensure in on_notify'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluent-plugin-windows-eventlog-0.2.1/lib/fluent/plugin/in_windows_eventlog.rb:184:in `on_notify'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluent-plugin-windows-eventlog-0.2.1/lib/fluent/plugin/in_windows_eventlog.rb:113:in `block (2 levels) in start'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluentd-1.2.5/lib/fluent/plugin_helper/timer.rb:80:in `on_timer'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/cool.io-1.4.6-x86-mingw32/lib/cool.io/loop.rb:88:in `run_once'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/cool.io-1.4.6-x86-mingw32/lib/cool.io/loop.rb:88:in `run'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluentd-1.2.5/lib/fluent/plugin_helper/event_loop.rb:93:in `block in start'
  2019-08-18 07:13:24 -0500 [error]: #0 C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.3.0/gems/fluentd-1.2.5/lib/fluent/plugin_helper/thread.rb:78:in `block in thread_create'
```

**Comment by qingling128** [comment]
Added some debugging and exception rescue to help investigate the issue: https://github.com/fluent/fluent-plugin-windows-eventlog/pull/34.

**Comment by cosmo0920** [comment]
Merged in #34. So, I'm closing this.
